client/a8c-for-agencies/sections/overview/body/hosting/index.tsx

```diff
@@ -1,5 +1,95 @@
+import page from '@automattic/calypso-router';
+import { useDispatch } from '@wordpress/data';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
+import Offering from 'calypso/a8c-for-agencies/components/offering';
+import { OfferingItemProps } from 'calypso/a8c-for-agencies/components/offering/types';
+import pressableIcon from 'calypso/assets/images/pressable/pressable-icon.svg';
+import WordPressLogo from 'calypso/components/wordpress-logo';
+import { recordTracksEvent } from 'calypso/state/analytics/actions/record';
+import './styles.scss';
+
+const A4A_HOSTING_PRESSABLE_MARKETPLACE_LINK = '/marketplace/hosting/pressable';
+const A4A_HOSTING_WPCOM_MARKETPLACE_LINK = '/marketplace/hosting/wpcom';
```

Comment:
[Non-blocker] I suggest we keep this in `/client/a8c-for-agencies/components/sidebar-menu/lib/constants.ts` since it will used in a lot of places.

Reply:
Oh, yes, sorry, forgot about this one.